src/Text/Pandoc/Writers/Shared.hs

```diff
@@ -213,6 +214,15 @@ fixDisplayMath (Para lst)
                          not (isDisplayMath x || isDisplayMath y)) lst
 fixDisplayMath x = x
 
+wrapMath :: Block -> Block
```

Comment:
This seems to be used only in the ODT writer, so it should go there.

Reply:
`wrapMath` has been removed.